lib/membrane_dashboard_web/live/dashboard_live.ex

```diff
@@ -67,14 +69,16 @@ defmodule Membrane.DashboardWeb.DashboardLive do
 
     # if we have no data assigned then request the full query
     if socket.assigns.data == empty_lists do
-      {:noreply, push_patch_with_params(socket, %{from: from, to: to})}
+      push_patch_with_params(socket, %{from: from, to: to}) |> noreply()
```

Comment:
Just one pipe operator is not recommended so I would rewrite it to 
```suggestion
      socket
      |> push_patch_with_params(%{from: from, to: to}) 
      |> noreply()
```